components/ads111x/ads111x.c

```diff
@@ -9,8 +9,10 @@
  *
  * BSD Licensed as described in the file LICENSE
  */
-#include "ads111x.h"
+
 #include <esp_log.h>
+#include "esp_idf_lib_helpers.h"
```

Comment:
I prefer to use `<>` instead of `""` in `#include` if header is part of another component.